lang/agda.bnf

```diff
@@ -24,7 +24,7 @@ nameDecl ::= IDENTIFIER {
 	mixin='org.ice1000.tt.psi.agda.AgdaNameDeclMixin'
 }
 
-module ::= KW_MODULE nameExp KW_WHERE
+module ::= KW_MODULE (nameExp | UNDERSCORE) KW_WHERE
```

Comment:
`nameExp` can already be an underscore: https://github.com/owo-lang/intellij-dtlc/blob/2c07f5ad9922f1b0be7eff909bb88f7bff02af86/lang/agda.bnf#L221

Reply:
ohh that's right

Reply:
Can you fix it?

Reply:
Pushed.